mountd/rpc_lsarpc.c

```diff
@@ -59,7 +59,7 @@ static struct policy_handle *lsarpc_ph_alloc(unsigned int id)
 	struct policy_handle *ph;
 	int ret;
 
-	ph = calloc(1, sizeof(struct policy_handle));
+	ph = g_try_malloc(sizeof(struct policy_handle));
```

Comment:
@neheb, @namjaejeon,

Just to make sure, is this actually intended? It effectively changes `calloc()` to `malloc()`, i.e. there is no zeroing. There is no mention of this in the PR or the commit message.

Reply:
this...looks like a typo.

Reply:
@neheb, @namjaejeon,

Would you fix this then? And, shouldn't there be a change from `free()` to `g_free()` for all the changed allocations in this PR? `g_free()` is not just a wrapper for `free()`.

Reply:
sure.

Reply:
@neheb, @namjaejeon,

> And, shouldn't there be a change from `free()` to `g_free()` for all the changed allocations in this PR? `g_free()` is not just a wrapper for `free()`.

Looks like this is an existing problem in this codebase. All the allocations from `g_` functions should be freed with `g_free()` and not `free()`. I can work on this change.